sdk/provisioner.go

```diff
@@ -107,6 +108,11 @@ func (out *ProvisionOutput) AddArgs(args ...string) {
 	out.CommandLine = append(out.CommandLine, args...)
 }
 
+// PrependArgs can be used to insert additional arguments at the beginning of the command line of the provision output.
+func (out *ProvisionOutput) PrependArgs(args ...string) {
+	out.CommandLine = slices.Insert(out.CommandLine, 1, args...)
```

Comment:
In general it's a bit weird to place flags as very first arg, but for `mysql`  this seems like the standard: https://dev.mysql.com/doc/refman/8.2/en/mysql.html

Reply:
Probably there are other tools out there as well that do that so I'm also fine with having this as a re-usable method.